public/blog/testrpc-is-now-ganache-cli.md

```diff
@@ -0,0 +1,25 @@
+# TestRPC is now Ganache CLI
+
+Have you been looking for the TestRPC recently and haven't been able to find anything about it?
+
+Don't worry, it's still here! It's just now known as **Ganache CLI**. You can get it here:
+
+```shell
+npm install -g ganache-cli
+```
+
+## A little history
+
+Prior to the existence of the TestRPC, one would have to deploy to a test network like Ropsten in order to see how your contract operates. The TestRPC solved that issue by providing a local blockchain where you could test your smart contracts without even needing to wait for your transactions to be mined into a block. 
+
+While initially under the auspices of the [EthereumJS project](https://github.com/ethereumjs/), it was designed by the Truffle team from the beginning, and today we remain its primary architect.
```

Comment:
Instead of `designed by the Truffle team from the beginning`, let's say `originally created by the Truffle team`. Might be splitting hairs, but some people might consider the (Javascript) codebase as having different beginnings, whereas the original TestRPC was written by me in python and was the first TestRPC in existence. 

Also, switch `architect` with `maintainers` to promote community effort.